DOCKER.md

```diff
@@ -0,0 +1,105 @@
+# Emissary In Docker
+
+### Intro
+
+Docker needs to be installed locally to build images. If developing on Linux, the maven plugin for Docker cannot be run 
+with ```sudo```. Please see the linux-postinstall page in the Docker documentation for instructions to manage Docker as 
+a non-root user. 
+
+### Build Emissary Docker Image
+
+Maven can be used to create the docker image. There is a profile that was created to run the docker image build that, by default,
+has been turned off. We'll need to add the docker profile, along with the dist profile, to trigger an assembly. From the
+project root, run the following maven command:
+
+```
+mvn clean package -Pdist,docker
+```
+
+Alternatively, we can use Docker directly. First run a full maven build and then run the ```docker build``` command:
+
+```
+mvn clean package -Pdist
+docker build -f contrib/docker/Dockerfile . -t emissary:latest
+```
+
+### Run Emissary with Docker
```

Comment:
This section is a bit confusing, because it uses "emissary" as the name for 3 different things, and that might not be obvious to people who are new to Emissary and/or Docker:

1. the name of the **Docker image** we build
2. the name of the **container** instance we launch from that image
3. the **command** we use within the container to launch the emissary server.

IMO, the commands in this section would be much more clear if we make two changes:

1. within each `docker run` command, specify the image using explicit `emissary:latest` syntax.  For Docker newbies, this will make it obvious that instance of "emissary" refers to the *image*; experienced Docker users should know the tag we're specifying could be omitted, but it shouldn't bother them.
2. use a container name that's not "emissary" (for instance, `--name em_container`).  This makes the removes ambiguity in any of the below commands. 

Thoughts?